src/signals/settings/users/Detail/components/UserForm/__tests__/UserForm.test.js

```diff
@@ -136,13 +189,33 @@ describe('signals/settings/users/containers/Detail/components/UserForm', () => {
     expect(radio2.checked).toBe(true);
   });
 
-  it('should check an unchecked checkbox', () => {
+  it('should check an unchecked role checkbox', async () => {
```

Comment:
`async` Is not necessary here.

Reply:
Oops! Normally ESLint warns me about this, will take a look later to enable this in our code base ;)